conformance/lang/expressions/checking-expression/checkpanic_expr.balt

```diff
@@ -0,0 +1,1050 @@
+    return y;
+}
+
+Test-Case: output
+Description: Test checkpanic-expr with literal values.
+Labels: error-type, optional-type, checkpanic, nil-type, nil-literal, decimal, DecimalFloatingPointNumber, DecimalNumber, float,
+        FloatingPointTypeSuffix, HexFloatingPointLiteral, HexIntLiteral, function-defn, function-call-expr, return-stmt,
+        value:toBalString(), int, additive-epr, unary-minus
+
+function init() {
+    error? x1 = fn1(); // @output 123
+                       // @output 9223372036854775807
+                       // @output -9223372036854775808
+                       // @output 1223617
+                       // @output -11256097
+
+    error? x2 = fn2(); // @output 12.3
+                       // @output 9.223372036854776E18
+                       // @output -92233.82036854776
+                       // @output 18.670913696289062
+                       // @output -171.75440979003906
+
+    error? x3 = fn3(); // @output 12.3
+                       // @output 9223372036854775807
+                       // @output -92233.82036854775807
+
+    error? x4 = fn4(); // @output
+                       // @output 9223372036854775807d
+
+    error? x5 = fn5(); // @output
+                       // @output ()
+}
+
+function fn1() returns error? {
+    int a1 = checkpanic 123;
+    io:println(a1);
+    int a2 = checkpanic 9223372036854775807;
+    io:println(a2);
+    int a3 = checkpanic (-9223372036854775807 - 1);
+    io:println(a3);
+    int a4 = checkpanic 0x12abC1;
+    io:println(a4);
+    int a5 = checkpanic -0xAbc121;
+    io:println(a5);
+}
+
+function fn2() returns error? {
+    float a1 = checkpanic 12.3;
+    io:println(a1);
+    float a2 = checkpanic 9223372036854775807;
+    io:println(a2);
+    float a3 = checkpanic (-92233.72036854775807 - .1);
+    io:println(a3);
+    float a4 = checkpanic 0x12.abC1;
+    io:println(a4);
+    float a5 = checkpanic -0xAb.c121;
+    io:println(a5);
+}
+
+function fn3() returns error? {
+    decimal a1 = checkpanic 12.3d;
+    io:println(a1);
+    decimal a2 = checkpanic 9223372036854775807d;
+    io:println(a2);
+    decimal a3 = checkpanic (-92233.72036854775807d - .1d);
+    io:println(a3);
+}
+
+function fn4() returns error? {
+    string a1 = checkpanic "";
+    io:println(a1);
+    string a2 = checkpanic "9223372036854775807d";
+    io:println(a2);
+}
+
+function fn5() returns error? {
+    () a1 = checkpanic ();
+    io:println(a1);
+    () a2 = checkpanic null;
+    io:println(a2.toBalString());
+}
+
+Test-Case: output
+Description: Test checkpanic-expr with string template expressions.
+Labels: checkpanic, string, function-defn, function-call-expr, return-stmt, optional-type, DecimalNumber, string-template-expr,
+        int
+
+function init() {
+    error? x1 = fn1(); // @output 224
+                       // @output ab
+                       // @output a
+}
+
+function fn1() returns error? {
+    string a1 = checkpanic (string`${12 + 212 - 0}`);
```

Comment:
Shall we also write some tests using checking expressions inside the string template?